content/en/boilerplates/crd-upgrade-123.md

```diff
@@ -0,0 +1,16 @@
+---
+---
+{{< warning >}}
+If upgrading CRDs via Helm from an Istio release 1.23 or older, you may encounter an error such as the following
+
+`Error: rendered manifests contain a resource that already exists. Unable to continue with update: CustomResourceDefinition "wasmplugins.extensions.istio.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata`
+
+You can resolve this with a one-time migration using the following `kubectl` commands:
+
+    {{< text syntax=bash snip_id=install_experimental_crds >}}
+    $ kubectl label crds -l chart=istio "app.kubernetes.io/managed-by=Helm"
+    $ kubectl annotate crds -l chart=istio "meta.helm.sh/release-name=istio-base"
```

Comment:
do we need a '`# replace with actual release name` comment here

Reply:
Yeah I dithered - we don't have such a comment for the actual chart install/upgrade tho - all these docs assume you are following our release naming instructions.

If you're confident enough to deviate from our docs as-written, you maybe should understand you need to do this etc etc.

I'll add it tho, I don't feel strongly.